lib/ProductOpener/Ingredients.pm

```diff
@@ -998,6 +1021,11 @@ fr => [
 'ne jamais recongeler un produit décongelé',
 'pensez au tri',
 'tenir à l\'abri',
+'Modes de pr(e|é)paration',
+'Mode de pr(e|é)paration',
+#'Pour votre santé',
```

Comment:
@stephanegigandet Should I activate those additional ones ?

Reply:
@teolemon : can you test it? use the CSV export, keep only the ingredients, and then grep your regexp. You will see everything that you will be cutting with your new regular expressions. If there are false positives (things that will be cut that should not be cut), then don't add it.